tests/suite/fixtures.py

```diff
@@ -727,6 +733,64 @@ def fin():
     )
 
 
+class TransportServerSetup:
+    """
+    Encapsulate Transport Server Example details.
+
+    Attributes:
+        name (str):
+        namespace (str):
+    """
+
+    def __init__(self, name, namespace):
+        self.name = name
+        self.namespace = namespace
+
+
+@pytest.fixture(scope="class")
+def transport_server_setup(
+        request, kube_apis, test_namespace
+) -> TransportServerSetup:
+    """
+    Prepare Transport Server Example.
+
+    :param request: internal pytest fixture to parametrize this method:
+        {example: virtual-server|virtual-server-tls|..., app_type: simple|split|...}
+        'example' is a directory name in TEST_DATA,
+        'app_type' is a directory name in TEST_DATA/common/app
+    :param kube_apis: client apis
+    :param test_namespace:
+    :return: TransportServerSetup
+    """
+    print(
+        "------------------------- Deploy Transport Server Example -----------------------------------"
+    )
+
+    # deploy global config
+    global_config_file = f"{TEST_DATA}/{request.param['example']}/standard/global-configuration.yaml"
+    gc_resource = create_gc(kube_apis.custom_objects, global_config_file, "nginx-ingress")
```

Comment:
We also have TransportServer for TLS Passthrough, which doesn't require a GlobalConfiguration, but requires the `-enable-tls-passthrough` cli arg.  would it be wise to assume we will have a different fixture for that?

Reply:
Good point. We should update the fixture/tests when we implement those tests. https://nginxsoftware.atlassian.net/browse/IND-23927